owncloudData/src/main/java/com/owncloud/android/data/spaces/db/SpacesDao.kt

```diff
@@ -135,6 +138,11 @@ interface SpacesDao {
     fun deleteSpaceForAccountById(accountName: String, spaceId: String)
 
     companion object {
+        private const val SELECT_ALL_SPACES = """
+            SELECT *
+            FROM ${ProviderMeta.ProviderTableMeta.SPACES_TABLE_NAME}
+        """
```

Comment:
This one will retrieve also the mountpoints, that probably we are not interested in, not sure

Reply:
Ok, for the moment I restricted it to retrieve only "project" and "personal" spaces. When we deal with the shares space, we will have to add it here